axon-server-connector/src/test/java/org/axonframework/axonserver/connector/processor/EventProcessorControlServiceTest.java

```diff
@@ -16,26 +16,55 @@
 
 package org.axonframework.axonserver.connector.processor;
 
+import io.axoniq.axonserver.grpc.ErrorMessage;
+import io.axoniq.axonserver.grpc.InstructionResult;
+import io.axoniq.axonserver.grpc.control.EventProcessorSegmentReference;
+import io.axoniq.axonserver.grpc.control.PlatformInboundInstruction;
+import io.axoniq.axonserver.grpc.control.PlatformOutboundInstruction;
+import org.axonframework.axonserver.connector.AxonServerConfiguration;
 import org.axonframework.axonserver.connector.AxonServerConnectionManager;
 import org.junit.jupiter.api.*;
 
+import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Consumer;
+import javax.annotation.Nonnull;
 
 import static io.axoniq.axonserver.grpc.control.PlatformOutboundInstruction.RequestCase.*;
 import static org.axonframework.axonserver.connector.TestTargetContextResolver.BOUNDED_CONTEXT;
 import static org.mockito.Mockito.*;
 
 class EventProcessorControlServiceTest {
 
+    public static final String GOOD_INSTRUCTION_ID = "goodInstructionId";
+    public static final String TRACKING_EVENT_PROCESSOR = "TEP";
```

Comment:
Nit: I think the test constants can be made `private`.

Reply:
Sure, right!